src/app/domain/models/account.py

```diff
@@ -0,0 +1,9 @@
+import uuid
+from django.db import models
+from app.domain.models import User
+
+
+class Account(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    user = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    balance = models.FloatField(null=True)
```

Comment:
Seria bom manter constraints mais rigorosas no DB para evitar perca de dados e inconsistências.

E não recomendo utilizar FloatField para lidar com dinheiro. É melhor utilizar o DecimalField.

```suggestion
    user = models.ForeignKey(User, on_delete=models.PROTECT)
    balance = models.DecimalField()
```